README.md

```diff
@@ -236,7 +236,6 @@ For more detailed changelogs, please check [the release changelogs][45].
 [33]: https://loophp-collection.rtfd.io
 [28]: https://loophp-collection.readthedocs.io/en/latest/pages/api.html
 [32]: https://loophp-collection.readthedocs.io/en/latest/pages/usage.html
-[46]: https://loophp-collection.readthedocs.io/en/latest/pages/examples.html
```

Comment:
This is a dead URL, I'm not sure if it pointed to a page that no longer exists or the page exists but it was just renamed, please advise.

Reply:
Thanks!